manifest.datestamps.json

```diff
@@ -1,11 +1,16 @@
 {
     "variables": {
       "CurrentReleaseDateStamp": "20241112",
+      "ltsc2025-DateStamp": "20241114",
```

Comment:
Update to whatever date this is planning to be released with.

Reply:
Added a new temporary date